helm/chaos-mesh/templates/controller-manager-service.yaml

```diff
@@ -11,12 +11,12 @@ metadata:
 spec:
   type: {{ .Values.controllerManager.service.type }}
   ports:
-#    - port: {{ .Values.controllerManager.service.port }}
-#      targetPort: http
-#      protocol: TCP
-#      name: http
+    - port: 8080
```

Comment:
I remove the `port` var here since I think it is useless. We can just use a hardcode 8080 here